chain/exchange/protocol.go

```diff
@@ -71,13 +73,15 @@ type validatedRequest struct {
 const (
 	Headers = 1 << iota
 	Messages
+	Validate
```

Comment:
I'm OK with this being part of the API of the exchange protocol (although it seems a bit of a stretch to me) but it shouldn't be part of the protocol itself, this can't go in the wire.

Reply:
yeah, the intention is to not go to the wire; it was the only way to pass the option to the api, I'll just make sure to clear the bit.

Reply:
done.